pyca/ingest.py

```diff
@@ -139,6 +140,10 @@ def control_loop():
                              .filter(RecordedEvent.status ==
                                      Status.FINISHED_RECORDING).first()
         if event:
+            delay = random.randint(config()['ingest']['delay_min'],config()['ingest']['delay_max'])
+            logger.info ("delaying %s second" % delay)
```

Comment:
```suggestion
            logger.info ("Delaying ingest for %s seconds", delay)
```

Minimal change but improves performance since the logger will only construct the string if necessary instead of them being always constructed. This will not really make a noticeable difference here but it's good to not start with bad habits ;-P